test/peer_SUITE.erl

```diff
@@ -156,3 +157,18 @@ association_test(_) ->
     ?assertError(invalid_association_signature, libp2p_peer:verify(InvalidPeer)),
 
     ok.
+
+stale_test(_) ->
+    #{secret := PrivKey1, public := PubKey1} = libp2p_crypto:generate_keys(ecc_compact),
+    SigFun1 = libp2p_crypto:mk_sig_fun(PrivKey1),
+
+    PeerMap = #{pubkey => libp2p_crypto:pubkey_to_bin(PubKey1),
+                listen_addrs => ["/ip4/8.8.8.8/tcp/1234"],
+                connected => [],
+                nat_type => static,
+                timestamp => erlang:system_time(millisecond) + 100},
```

Comment:
what's this timestamp actually do?

Reply:
This was just a test out of curiosity will remove